public/app/features/apiserver/server.ts

```diff
@@ -20,9 +21,14 @@ export class ScopedResourceServer<T = object, K = string> implements ResourceSer
   readonly url: string;
 
   constructor(gvr: GroupVersionResource, namespaced = true) {
+    const configUrl = config.bootData.settings.apiServerUrlOverrides?.find(
+      (override) =>
+        override.group === gvr.group && override.version === gvr.version && override.resource === gvr.resource
+    )?.url;
+
     const ns = namespaced ? `namespaces/${config.namespace}/` : '';
 
-    this.url = `/apis/${gvr.group}/${gvr.version}/${ns}${gvr.resource}`;
+    this.url = configUrl ?? `/apis/${gvr.group}/${gvr.version}/${ns}${gvr.resource}`;
   }
```

Comment:
Shouldn't this constructor be implemented in a base class if we are aiming at building a reusable primitive for a resource server?

Reply:
Ah, wait this is actually base class, I have thouth it's `Scopes`ResourceServer, but it's `Scoped`ResourceServer. Why is it "scoped" acually? think as long as we don't have "regular" ResourceServer, there is no point of naming it a `ScopedResourceServer`. 

But you know, naming is hard :D

Reply:
I think it's scoped because it's actually an instance of a client for a specific group/version/resource 🤔 . But yeah, no strong preferences towards it.